pkgs/by-name/th/the-way/package.nix

```diff
@@ -19,8 +19,6 @@ rustPlatform.buildRustPackage rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  useNextest = true;
-
   postInstall = ''
```

Comment:
Also, it would be good to make this conditional, see #308283

Reply:
useNextest = true; is fine, I was thinking of the postInstall part, where it runs the compiled binary.
That doesn't work for cross-compiling

Reply:
My bad :see_no_evil: I'll fix it